French translation update [Issue #333]

BRUCELLA2 opened the pull request:
## Status
**READY**

## Migrations
NO

## Description
French translation update

BRUCELLA2 committed:
Added French language translations [Issue #333]

Added some french translations

Fix French language translations [Issue #333]

mcpierce commented:
@BRUCELLA2 Can you rebase this PR so that it's just the translation, please?

BRUCELLA2 commented:
> @BRUCELLA2 Can you rebase this PR so that it's just the translation, please?

Yes, i can :+1:

mcpierce commented on comixed-frontend/src/assets/i18n/fr/comics.json:
```diff
@@ -181,36 +181,35 @@
       "column": {
-        "blocked-status": "{blocked, select, true{Yes} other{No}}",
-        "deleted-status": "{deleted, select, true{Yes} other{No}}"
+        "blocked-status": "{bloquée, sélectionner, true{Oui} other{Non}}",
```
In this and the following line, the keyword "select" was translated, and needs to be reverted. Also the variables "blocked" and "deleted" were translated and need to be reverted.

BRUCELLA2 replied:
fixed

mcpierce commented on comixed-frontend/src/assets/i18n/fr/comics.json:
```diff
@@ -181,36 +181,35 @@
-      "blocked": "Click to {blocked, select, true{unblock} other{block}} this page type...",
-      "deleted": "Click to {deleted, select, true{undelete} other{delete}} this page..."
-    }
+      "blocked": "Cliquer pour {bloquer, sélectionner, true{débloquer} other{bloquer}} le type de cette page...",
```
Here and in the following line the key word "select" was translated and needs to be reverted. Also the variables "blocked" and "deleted" were translated and need to be reverted.

BRUCELLA2 replied:
fixed

mcpierce commented on comixed-frontend/src/assets/i18n/fr/library.json:
```diff
@@ -1,36 +1,36 @@
+      "delete-files-warning-header": "Suppression des fichiers physiques",
+      "delete-files-warning-message": "La suppression des fichiers physiques ne peut pas être annulée. Êtes-vous sûr de vouloir le faire ?",
+      "confirmation-header": "Organiser la bibliothèque",
+      "confirmation-message": "Êtes-vous sûr de vouloir {effacer les fichiers physiques, sélectionner, true {effacer les fichiers dex bande dessinée marqués pour suppression et} autres { et}} organiser la bibliothèque ?"
```
The variable "deletePhysicalFiles" was translated and needs to be reverted.

The keyword "other" was translated here and needs to be reverted. Also there shouldn't be a space between the keyword and the opening curly brace.

BRUCELLA2 replied:
fixed

mcpierce reviewed (changes requested):
Found a few issues with keywords and variable names being translated. I also saw an entry that wasn't translated in comics.json:

```"max-records": "Max Records To Fetch"```

Could you translate that line as well? And please be sure to test the translation locally.

BRUCELLA2 committed:
Update French language translations [Issue #333]

Fix Update French language translations [Issue #333]

Fix text and update French language translations [Issue #333]

mcpierce commented:
@BRUCELLA2 Is this rebased on the latest develop?
